bzip2/tests/test.pester.ps1

```diff
@@ -0,0 +1,15 @@
+param (
+    [Parameter()]
+    [string]$PackageIdentifier = $(throw "Fully qualified package identifier must be given as a parameter.")
+)
+
+$PackageVersion = $PackageIdentifier.split('/')[2]
+Describe "core/bzip2" {
+    Context "bzip2" {
+        $OutputVariable = (hab pkg exec $pkg_ident bzip2 --help *>&1 | Out-String)
+        $OutputVariable -match "(?<=bzip2, a block-sorting file compressor.  Version )([^,]*)"
```

Comment:
Same question as for Linux, is it possible to match for `Version ([0-9]+\.[0-9]+\.[0-9+])` and not depend on the wording of the help test?

Reply:
Same here.  Fixed in #2741.